gemma-core/src/main/resources/ubic/gemma/model/expression/designElement/CompositeSequence.hbm.xml

```diff
@@ -18,7 +18,7 @@
          <column name="DESCRIPTION" not-null="false" unique="false" sql-type="text"/>
       </property>
       <many-to-one name="biologicalCharacteristic" class="ubic.gemma.model.genome.biosequence.BioSequence"
-                   lazy="false" fetch="join">
+                   lazy="false" fetch="select">
```

Comment:
I've already backported this in the 1.31.1 0ccfe93a72d8ef0087d0440f8665b79d7f3dc5f3